### PR TITLE
add cachy-chroot

### PR DESCRIPTION
**Pull request by SoulHarsh007**

### CODE REVIEW

File: cachy-chroot/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Maintainer: SoulHarsh007 <harsh.peshwani@outlook.com>
+
+pkgname=cachy-chroot
+pkgver=1.0.0
+pkgrel=1
+pkgdesc='Chroot helper for CachyOS'
+arch=('x86_64' 'x86_64_v3')
```
Comment (ptr1337) [inline cachy-chroot/PKGBUILD:7]: I think we can remove this. We are using nowadays --ignorearch or overwrite it with pacman PACKAGEARCH feature

**Review by ptr1337 — APPROVED** [review]
Looking good, small comment

File: cachy-chroot/PKGBUILD (hunk 1 above)
Comment (vnepogodin) [reply cachy-chroot/PKGBUILD:7]: Its anyways cannot be build with v3 carch

**Review by vnepogodin — CHANGES_REQUESTED** [review]
You cannot use any for compiled binary

File: cachy-chroot/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+pkgrel=1
+pkgdesc='Chroot helper for CachyOS'
+arch=('any')
+groups=('cachyos')
+license=(BSD-3-Clause)
+url="https://github.com/CachyOS/cachy-chroot"
+depends=('arch-install-scripts' 'btrfs-progs' 'util-linux')
+makedepends=('git' 'cargo' 'lld' 'clang' 'llvm')
+source=("${pkgname}-${pkgver}.tar.gz::${url}/archive/${pkgver}.tar.gz")
+b2sums=('SKIP')
```
Comment (vnepogodin) [inline cachy-chroot/PKGBUILD:14]: Also update checksums

**Review by vnepogodin — CHANGES_REQUESTED** [review]

File: cachy-chroot/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+pkgver=1.1.0
+pkgrel=1
+pkgdesc='Chroot helper for CachyOS'
+arch=('any')
+groups=('cachyos')
+license=(BSD-3-Clause)
+url="https://github.com/CachyOS/cachy-chroot"
+depends=('arch-install-scripts' 'btrfs-progs' 'util-linux')
+makedepends=('git' 'cargo' 'lld' 'clang' 'llvm')
+source=("${pkgname}-${pkgver}.tar.gz::${url}/archive/${pkgver}.tar.gz")
```
Comment (ptr1337) [inline cachy-chroot/PKGBUILD:13]: If tag is signed, then use signed .tar.gz

File: cachy-chroot/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+b2sums=('SKIP')
+provides=('cachy-chroot')
+options=(strip)
```
Comment (ptr1337) [inline cachy-chroot/PKGBUILD:16]: Strip is enabled by default

File: cachy-chroot/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+pkgdesc='Chroot helper for CachyOS'
+arch=('any')
+groups=('cachyos')
+license=(BSD-3-Clause)
+url="https://github.com/CachyOS/cachy-chroot"
+depends=('arch-install-scripts' 'btrfs-progs' 'util-linux')
+makedepends=('git' 'cargo' 'lld' 'clang' 'llvm')
+source=("${pkgname}-${pkgver}.tar.gz::${url}/archive/${pkgver}.tar.gz")
+b2sums=('SKIP')
+provides=('cachy-chroot')
```
Comment (ptr1337) [inline cachy-chroot/PKGBUILD:15]: pkgname already provides cachy-chroot, not required

**Review by vnepogodin — APPROVED** [review]
lgtm

**Pull request merged**